tests/js/spec/utils/performance/suspectSpans/spanOpsQuery.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import {mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import EventView from 'app/utils/discover/eventView';
+import SpanOpsQuery from 'app/utils/performance/suspectSpans/spanOpsQuery';
+import {
+  SpanSortOthers,
+  SpanSortPercentiles,
+} from 'app/views/performance/transactionSummary/transactionSpans/types';
+
+describe('SuspectSpansQuery', function () {
+  let eventView, location;
+  beforeEach(function () {
+    eventView = EventView.fromSavedQuery({
+      id: '',
+      name: '',
+      version: 2,
+      fields: [...Object.values(SpanSortOthers), ...Object.values(SpanSortPercentiles)],
```

Comment:
Is there a reason to even pass in fields here if we're just checking that the api gets called at all?

Reply:
Yes, because the `eventView` checks the sort against the selected fields or and silently discards the sort if it fails validation.